docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -43,6 +43,12 @@ PUT /_cluster/settings
 -------------------------------
 // CONSOLE
 
+NOTE: User-defined cluster metadata is not intended to store sensitive or
```

Comment:
Consider making this an `IMPORTANT` section instead of a `NOTE` section, given that it is about sensitive information.